source/postprocess/visualization.cc

```diff
@@ -573,6 +573,10 @@ namespace aspect
           vtk_flags.cycle = this->get_timestep_number();
           vtk_flags.time = time_in_years_or_seconds;
 
+#if DEAL_II_VERSION_GTE(9,1,0)
+          vtk_flags.write_higher_order_cells = write_higher_order_output;
```

Comment:
Shouldn't this be dependent on whether you set the option `Write higher order output` to true?

Reply:
It is. `write_higher_order_output` is only true if it is set in the input file, otherwise it is false and nothing changes.

Reply:
ah, I now see. I overlooked it :(